src/osbuild-manifests/coreos.osbuild.aarch64.mpp.yaml

```
@@ -22,6 +25,18 @@ mpp-vars:
   boot_fs_label: boot
   root_fs_uuid: 910678ff-f77e-4a7d-8d53-86f2ac47a823
   root_fs_label: root
+  # Set the buildroot string to use for most operations here. We create
+  # the buildroot from the target OSTree contents so we have version
+  # matches. Unfortunately for FCOS there is no python so we can't
+  # really use FCOS as the buildroot so we'll set it to "" so that
+  # the host (COSA) gets used as the buildroot there.
+  buildroot:
+    mpp-if: osname == 'rhcos'
```

Not related to the current CI failure, but shouldn't this be

```suggestion
    mpp-if: osname in ['rhcos', 'scos']
```

?